code/game/objects/structures/false_walls.dm

```diff
@@ -426,6 +426,10 @@
 	icon_state = ""
 	mineral = "gold"
 
+/obj/structure/falsewall/gold/closed
```

Comment:
Is this really necessary? Can't you just manually set these vars in StrongDMM instead of making a new child type?